### DISCUSSION TITLE
Is it possible to expire a table row?

### DISCUSSION THREAD

**Discussion by alucardu** [prisma/prisma#13098] [Q&A] [answered]
I want to implement a feature for user to reset their password. 

For this I want to create a unique id that will be stored in my Postgres DB and it will be send to the user's email-address (in the format of a link). When the user follows the link the client will check if the id in the url matches any id in the forgotDB table (and perhaps also match the email address). If those things match the user can update his password field. One issue I see here is that I don't want the id's in the database to be stored infinitely. If something were to happen someone could abuse those id's and email addresses (maybe I shouldn't store the email address with the id ...). Is it possible for Prisma to accommodatie this? For example expire a row after 24 hours? 

ps. First time I do something like this, so if there's any feedback on how I approach this problem, please let me know.

**Comment by nurul3101** [comment, marked as answer]
Hey @alucardu 👋 

This is something which could be achieved by a cron which could run at specific intervals and delete the expired records. 
This isn't natively supported by Prisma to automatically delete records.

**Reply by alucardu** [reply]
Thanks I realized this was a limitation of Postgres and not of Prisma, but there's a nice `pg_cron` plugin that fixed everything :)

**Reply by nurul3101** [reply]
That's amazing! Glad to hear that this is fixed 🙌

**Comment by Peace-Ishimwe** [comment]
How did you use the pg_cron to solve the issue

**Reply by alucardu** [reply]
I don't have the exact cron job on hand anymore, but > https://supabase.com/docs/guides/database/extensions/pg_cron should explain it. What I did was run a cron job to clear all rows in a table.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
